### DISCUSSION TITLE
Regarding the project: Modularization of Components

### DISCUSSION THREAD

**Discussion by psankhe28** [camicroscope/GSOC#31] [Q&A]
There are many ways in which we can achieve modularization of web components. Is there any way that is preferred? Or are we free to choose and create the proposal accordingly?

**Comment by nanli-emory** [comment]
Hi @psankhe28, every problem can be solved in multiple ways. You can put your idea and give a explain why you choose this way over another. The proposal is not only a proposal but we also want to know your thoughts through these works, such as the code challenge.

**Reply by psankhe28** [reply]
By multiple ways, I meant by using some other already built packages related to this.

**Reply by nanli-emory** [reply]
sorry, we don't encourage you to reinvent the wheel. You should mention what packages you prefer and what reason you choose them.

**Reply by psankhe28** [reply]
Yeah, I have mentioned in the proposal in detail.

**Comment by psankhe28** [comment]
@nanli-emory If possible, could you review my proposal for the project "Modularization of Components" and  "Collection and Study Management"? Could you please go through it once and provide me with feedback? Thanks.
